src/rewrite_quantization.cpp

```diff
@@ -62,9 +64,22 @@ void apply_quantizelinear(module& m, instruction_ref ins)
         max_quant = qt.max();
         min_quant = qt.min();
     });
-    auto s        = add_zero_point->get_shape();
-    auto min_arg  = m.add_literal(literal{shape{s.type()}, {min_quant}});
-    auto max_arg  = m.add_literal(literal{shape{s.type()}, {max_quant}});
+    auto s = add_zero_point->get_shape();
+    instruction_ref min_arg;
+    instruction_ref max_arg;
+
+    if(enabled(MIGRAPHX_ENABLE_CK{}))
```

Comment:
I think it would be better to have a separate env variable to enable these changes, like `MIGRAPHX_ENABLE_CK_WORKAROUNDS` as this is a pessimization.

Reply:
But without it CK will crash. Wouldn't it be better to have them on by default whenever CK is enabled?

Reply:
The problem is how do we test if it works? Also, if mlir is enabled along with ck then we dont want to do this pessimization either. I think it would be better to disable CK fusions for int8 when there are scalars. Need to run numbers, but I think MLIR will be much faster then CK for all int8 cases.